microcosm_postgres/factories/engine_strategy.py

```diff
@@ -0,0 +1,54 @@
+from microcosm.scoping.proxies import ScopedProxy
+
+
+def unwrap(bind):
+    return bind.__component__ if isinstance(bind, ScopedProxy) else bind
+
+
+class EngineRoutingStrategy:
```

Comment:
The basic strategy is to use the singleton `graph.postgres` engine as the bind.

I chose not to use an ABC here because there was no real need.